pkg/cloudprovider/suite_test.go

```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/aws/aws-sdk-go/service/ssm"
+	opstatus "github.com/awslabs/operatorpkg/status"
```

Comment:
nit: Why alias this?

Reply:
there's another import that says status

Reply:
Where?

Reply:
https://github.com/aws/karpenter-provider-aws/pull/6195/files#diff-25a0b7de23b450e40e53e28c267ee5bd513470d4e8dfeb72893106fc19e5a5f3R42

Reply:
L 42

Reply:
Ah, got it. I think I'd still prefer aliasing the other thing as `nodeclassstatus` and not aliasing this. This is non-blocking though.